docs/NooBaaNonContainerized/ConfigFileCustomizations.md

```diff
@@ -478,6 +478,34 @@ Warning: After setting this configuration, NooBaa will skip schema validations a
     3. systemctl restart noobaa
     ```
 
+### 33. Notification space monitor frequency flag -
+* <u>Key</u>: `NOTIFICATION_REQ_PER_SPACE_CHECK`
+* <u>Type</u>: Number
+* <u>Default</u>: 0
+* <u>Description</u>: Free space check will run per this many notifications. 0 to disable.
+* <u>Steps</u>:
+    ```
+    1. Open /path/to/config_dir/config.json file.
+    2. Set the config key -
+    Example:
+    "NOTIFICATION_REQ_PER_SPACE_CHECK": 1000
+    3. systemctl restart noobaa
```

Comment:
Are you sure you need to restart the service for this?

Reply:
How else config.js will be reloaded? Is there a way to reload it while endpoint is running?

Reply:
I'm just asking because there are cases in this configs where it was not mentioned.